nss-lean-anteaters/nutshell
# login user form

## Link to Ticket
Closes # [https://github.com/nss-lean-anteaters/nutshell/issues/30]()

## Description of Proposed Changes

- Added fields for logging in an existing user

## Steps to Test

Outline the steps to test

```
git fetch --all
git checkout dv-login-view
code .
```
1. check grunt on src
1. open hs-o on dist
1. click on "login" button
1. The body of the DOM should be cleared (Ronnie's name will disappear)
1. A form should be added for Username and email and "login" button
1. Add information for username and email and click "comp reg" button - should see console log of those values
1. Add info for username and put nothing in email field and click "login" button - should see console log of email and an alert telling you to fill in both fields (you can try with the email field blank as well)
 

## Impacted Areas in Application

List general components of the application that this PR will affect:

* Main page of app will now include a small form to login an existing user

## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
